### PR TITLE
Prototype sitemap

### PR DESCRIPTION
**Pull request by sophiedrake**
https://triadmoj.atlassian.net/browse/DIV-204

Assumed this is only needed for the latest version.

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -159,6 +159,34 @@ app.get('/check-session', function (req, res) {
 app.use('/v1', v1routes);
 app.use('/v2', v2routes);
 
+// Create sitemap
+app.get('/sitemap', function (req, res) {
+  var rootPath = './app/views/v2';
+  var walk    = require('walk');
+  var files   = {};
+  var walker  = walk.walk('./app/views/v2');
```
Comment (lukaszdrynski) [inline server.js]: you can use rootPath variable here

File: server.js [hunk 2]
```diff
@@ -159,6 +159,34 @@ app.get('/check-session', function (req, res) {
...
+
+  walker.on('file', function(root, stat, next) {
+    var directory = root.replace('./app/views/v2', '');
```
Comment (lukaszdrynski) [inline server.js]: and here

**Comment by sophiedrake** [comment]
updated

**Comment by lukaszdrynski** [comment]
👍

**Pull request merged**